app/controllers/users_controller.rb

```diff
@@ -46,12 +46,11 @@ def destroy
   private
 
   def user_params
-    params.require(:user).permit(
-                                  :name,
-                                  :username,
-                                  :email,
-                                  :password,
-                                  :password_confirmation
-                                  )
+    params.require(:user).permit(:name,
+                                 :username,
+                                 :email,
+                                 :password,
+                                 :password_confirmation
+                        .r        )
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Unnecessary spacing detected.<br>Space inside parentheses detected.